test/libsolidity/semanticTests/cleanup/cleanup_bytes_types_shortening.sol

```diff
@@ -11,5 +11,7 @@ contract C {
     }
 }
 
+// ====
+// compileViaYul: also
 // ----
-// f() -> "\xff\xff\xff\xff"
+// f() -> "����"
```

Comment:
wat

Reply:
Hmmm... interesting... I didn't even look at that - those were auto-updated by ``isoltest`` and apparently that changed the encoding :-D.